build.gradle

```diff
@@ -369,4 +369,12 @@ tasks.register("listRepositories") {
         println "Repositories:"
         project.repositories.each { println "Name: " + it.name + "; url: " + it.url }
    }
-}
\ No newline at end of file
+}
+
+tasks.register("writeGithubBuildDetails", GithubDetailsTask) {
+    text = """### Test result files
+
+- [Unit Tests (Debug)](/reports/tests/testDebugUnitTest/index.html)
```

Comment:
Won't it be less confusing and simpler to set this to `https://firefoxci.taskcluster-artifacts.net/$taskId/0/public` and just replace $taskId with a real one in your task? The regex you have is clever, but that's almost never a good attribute of code :)

Reply:
I'm trying to avoid the long URL here. Maybe something like `- [Unit Tests (Debug)]({reportsUrl}/reports/tests/testDebugUnitTest/index.html)`? Would allow for shorter URL with simpler replace semantics.

Reply:
Yeah, that works too!